R/load_pop_data.R

```diff
@@ -107,54 +104,51 @@ set_age <- function(object, ...){
 }
 
 #' @export
-set_age.pop_data <- function(object, age = 'Age', standardize, ...){
+set_age.pop_data <- function(object, age = 'Age', standardize = TRUE, ...){
 
-  if(standardize)
```

Comment:
we want to find the right column first, and possibly standardize second; I moved this code to later in this function.